sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -254,6 +254,18 @@ case class StaticInvoke(
     returnNullable: Boolean = true,
     isDeterministic: Boolean = true) extends InvokeLike {
 
+  def this(
```

Comment:
Add a comment here explaining why we need this? Without context, this looks a bit redundant.

Reply:
Sure added comments.